cwl/src/main/scala/wdl4s/cwl/TypeAliases.scala

```diff
@@ -6,6 +6,9 @@ import eu.timepit.refined._
 
 trait TypeAliases {
 
+  //TODO: This can parse anything :(  Need to figure out how to allow that (likely via custom parsing)
```

Comment:
ToL: I guess this is a personal reminder, but just to say, I don't really understand this TODO